daemon/graphdriver/btrfs/btrfs.go

```diff
@@ -51,29 +53,35 @@ func Init(home string, options []string) (graphdriver.Driver, error) {
 	return graphdriver.NaiveDiffDriver(driver), nil
 }
 
+// Driver contains information about the filesystem mounted.
 type Driver struct {
+	//root of the file system
 	home string
 }
 
+// String prints the name of the driver (btrfs).
 func (d *Driver) String() string {
 	return "btrfs"
 }
 
+// Status returns current driver information.
```

Comment:
How is a slice of statuses not plural?

Based on the method name, we know it returns status. This is a specific implementation for btrfs, so certain statuses may be returned. What are they? What do they mean? How can they be used? Are there caveats?

The proposed documentation provides absolutely no benefit to the reader of the code.